05-transform.Rmd

```diff
@@ -642,7 +642,7 @@ seine_centroid = st_centroid(seine)
 Centroids could be useful to represent more complex objects - lines and polygons, for example to calculate distances between centers of polygons.
 They are also often used as places where polygons or lines labels are put. 
 However, it is important to know that centroids could be located outside of the given object, e.g. in cases of irregular shaped polygons or lines.
-Examples of this could be seen on the right plot on Figure \@ref(fig:centr).
+Examples of this can be seen on the right plot on Figure \@ref(fig:centr).
```

Comment:
Heads-up @Nowosad can is usually more appropriate than could 👍

Reply:
Thanks @katygregg and @Robinlovelace - I'll work on fixing my brain on can/could;)